config/initializers/curation_concerns.rb

```diff
@@ -9,3 +9,13 @@
   end
 end
 
+# We're actually overriding from the sufia-set Sufia::FileSetPresenter, to our
+# own. to_prepare seems necessary to make it stick in dev.
+Rails.application.config.to_prepare do
+  CurationConcerns::MemberPresenterFactory.file_presenter_class = CHF::FileSetPresenter
```

Comment:
Hmm... I think they can! Not sure if that makes sense for just setting a class variable... but if you think it does, I'm happy to roll with it!